src/Databases/DatabaseFilesystem.cpp

```diff
@@ -141,19 +143,18 @@ StoragePtr DatabaseFilesystem::getTableImpl(const String & name, ContextPtr cont
     if (!checkTableFilePath(table_path, context_, throw_on_error))
         return {};
 
-    String format = FormatFactory::instance().getFormatFromFileName(table_path, throw_on_error);
+    auto format = FormatFactory::instance().getFormatFromFileName(table_path, throw_on_error);
     if (format.empty())
         return {};
 
-    /// If the file exists, create a new table using TableFunctionFile and return it.
-    auto args = makeASTFunction("file", std::make_shared<ASTLiteral>(table_path), std::make_shared<ASTLiteral>(format));
+    auto astFunctionPtr = makeASTFunction("file", std::make_shared<ASTLiteral>(table_path), std::make_shared<ASTLiteral>(format));
```

Comment:
```suggestion
    auto ast_function_ptr = makeASTFunction("file", std::make_shared<ASTLiteral>(table_path), std::make_shared<ASTLiteral>(format));
```
we use a different style

Reply:
Done!